SPECS/libmodulemd/libmodulemd.spec

```diff
@@ -32,47 +33,55 @@ C Library for manipulating module metadata files
 Summary:        Header and development files for libmodulemd
 Requires:       libyaml-devel
 Requires:       %{name} = %{version}-%{release}
+
 %description    devel
 It contains the libraries and header files.
 
 %prep
-%setup -q -n modulemd-%{version}
-%patch1 -p1
+%autosetup -p1 -n modulemd-%{version}
 
 %build
-meson -Dprefix=%{_prefix} -Ddeveloper_build=false -Dbuild_api_v1=true -Dbuild_api_v2=false \
-       -Dwith_py3_overrides=true -Dwith_py2_overrides=false api1
-cd api1
-ninja
+%meson \
+    -Ddeveloper_build=false \
+    -Dbuild_api_v1=true \
+    -Dbuild_api_v2=true \
+    -Dwith_py3_overrides=true \
+    -Dwith_py2_overrides=false \
+    -Dwith_docs=false \
```

Comment:
Shouldn't this also now depend on the value of `%{with docs}`?

Reply:
Yep, good catch! Fixed.